HW04.Rmd

```diff
@@ -68,54 +126,114 @@ Assume that:
 
 
 ### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
 
-```{r police}
+```{r police and age}
+police_gun_deaths <- gun_deaths %>%
+    select(police, age, race) %>%
+    filter(age != "Not Available") %>%
+    mutate(age = as.numeric(age)) %>%
+    group_by(police)
+
+ggplot(police_gun_deaths, mapping = aes(x = factor(police), y = age)) +
+    geom_boxplot(fill = "maroon") +
+    labs(title = "Age Distribution of Gun Death Victims caused by Police Involvement ", 
+          x = "Police Involvement", 
+          y = "Age (Years)") +
+    theme(plot.title = element_text(hjust = 0.5)) +
+    scale_x_discrete(labels=c("Uninvolved", "Involved"))
+```
 
+```{r police and race}
+new_police_gun_deaths <- gun_deaths %>%
+    select(police, age, race) %>%
+    filter(age != "Not Available") %>%
+    mutate(age = as.numeric(age)) %>%
+    group_by(police, race) %>%
+    count(police) %>%
+    group_by(police)
+
+ggplot(new_police_gun_deaths, mapping = aes(x = factor(police), y = n)) +
+    geom_col(aes(fill = race)) +
+    labs(title = "Race Distribution of Gun Death Victims caused by Police Involvement ", 
+          x = "Police Involvement", 
+          y = "Race") +
+    scale_x_discrete(labels=c("Uninvolved", "Involved"))
+
+#This graph looks slightly off, I am not sure if the values for the 'involved' category is right. Will have to double-check!
 ```
 
 Think about performing stats such as a t.test to see if differences you are plotting are statistically significant
 
 ```{r stats?}
 
+age_police_uninvolved <- police_gun_deaths %>%
+    filter(police == 0)
+age_police_involved <- police_gun_deaths %>%
+    filter(police == 1)
 
+t.test(age_police_uninvolved$age, age_police_involved$age)
 ```
+**Based on this t-test, the difference in age is statistically significant. 
 
 
-```{r police_2}
-
-```
-
-```{r stats_2?}
-
-
-```
-
 
 Finally, all together now:
 ```{r police_3}
-
+police_gun_deaths <- gun_deaths %>%
+    select(police, age, race) %>%
+    filter(age != "Not Available") %>%
+    mutate(age = as.numeric(age)) %>%
+    group_by(police)
+    
+ggplot(police_gun_deaths, mapping = aes(x = factor(police), y = age, shape = race, color = race)) +
+    geom_point () +
+    facet_wrap(.~race, ncol = 5) +
+    labs(title = "Intersection of Police Involvement, Age and Race in US Gun Deaths", 
+          x = "Police Involvement", 
+          y = "Age (Years)") +
+    scale_x_discrete(labels=c("Uninvolved", "Involved")) +
+    theme(axis.text.x = element_text(angle = 45, vjust = 0.6))
 ```
-
+*I don't think that this is the best graph to represent the intersection of the three variables. Maybe I should have done percentages of race and create a pie chart, I am not sure. Can we even make a pie chart using R? This is something I shall look into!

```

Comment:
I made a graph similar to this at first and then turned it into a boxplot to look more at the average age for each group which I think might be slightly more informative than the version you currently have here.  I guess both technically plot the same information but I think the boxplot is a litter easier to draw some type of conclusion from in this instance.